tasks/linter.py

```diff
@@ -106,6 +110,7 @@ def go(
     golangci_lint_kwargs="",
     headless_mode=False,
     include_sds=False,
+    platform=None,  # noqa: U100. Used by the run_on_devcontainer decorator
```

Comment:
To reviewers: this is the only thing we need to add to other tasks so they can benefit from it